src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java

```diff
@@ -0,0 +1,113 @@
+package com.commercetools.sync.inventories.utils;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.commands.updateactions.ChangeQuantity;
+import io.sphere.sdk.inventory.commands.updateactions.SetExpectedDelivery;
+import io.sphere.sdk.inventory.commands.updateactions.SetRestockableInDays;
+import io.sphere.sdk.inventory.commands.updateactions.SetSupplyChannel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+
+/**
+ * This class provides static utility methods for building update actions related to inventories.
+ */
+public final class InventoryUpdateActionUtils {
+
+    private InventoryUpdateActionUtils() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Compares the {@code quantityOnStock} values of a {@link InventoryEntry} and {@link InventoryEntryDraft}
+     * and returns a {@link Optional} of update action, which would contain the {@code "changeQuantity"}
+     * {@link UpdateAction}. If both {@link InventoryEntry} and {@link InventoryEntryDraft} have the same
+     * {@code quantityOnStock} values, then no update action is needed and empty optional will be returned.
+     * If {@code quantityOnStock} from {@code newEntry} is {@code null} then it is assumed to represent empty stock
+     * (like it would be 0L value).
+     *
+     * @param oldEntry the inventory entry that should be updated
+     * @param newEntry the inventory entry draft which contains new quantity on stock
+     * @return optional containing update action or empty optional if quantities on stock are indentical or quantity
```

Comment:
I forgot to update after changing. It's done now f804ac7504e18754867eb21e06ef0691f694f1b2